go/src/socialapi/workers/cmd/presence/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"socialapi/config"
+	"socialapi/workers/presence"
+
+	"github.com/koding/runner"
+)
+
+var (
+	name = "Presence"
+)
+
+func main() {
+	r := runner.New(name)
+	if err := r.Init(); err != nil {
+		fmt.Println(err)
```

Comment:
shouldn't it exit with non-success code when init fails? `log.Fatal` or `os.Exit`